### ISSUE TITLE
Enabling keyboard navigation prevents form dropdowns in Fancybox from being navigable with up/down keys

### ISSUE THREAD

**Issue by ginaNoticed** [fancyapps/fancybox#2213]
**Describe the bug**
I have a form I am loading into Fancybox. With keyboard navigation enabled, I am unable to cycle through options in an HTML select using the up/down keys. 

**To Reproduce**
1. Option 'keyboard: true'
2. Load content with form dropdown (select)
3. Attempt to change value of dropdown using keyboard up/down arrows
4. Nothing happens

**Expected behavior**
Should be able to change the value of the dropdown using the keyboard up/down arrows

**Additional context**
This is an accessibility issue. I assume the keyboard navigation option was added for accessibility reasons, so I'd like to keep it on, but it creates an additional accessibility issue with using the keyboard to perform tasks.

(Fancybox 3)
